configs/llama-v3-8b/infra/inference.config.json

```diff
@@ -0,0 +1,8 @@
+{
+  "SUBSCRIPTION_ID": null,
+  "RESOURCE_GROUP_NAME": null,
+  "STORAGE_ACCOUNT_NAME": null,
+  "FILE_SHARE_NAME": null,
+  "ACA_APP_NAME": null,
+  "ACA_APP_ENDPOINT": null
```

Comment:
We have recently updated both the `./infra` folder and the README file. Would you be able to sync the latest changes from the `ai-toolkit` branch in the `llama-v2-7b` folder?

Reply:
I've pulled the latest changes and updated the files in llama-v3-8b to match the rest of the updated model templates. Can you take another look and let me know if I missed anything?